src/main/java-templates/com/easypost/package-info.java

```diff
@@ -1,9 +0,0 @@
-/**
```

Comment:
No, the original error was because this existed, but as part of the build process, it autogenerates this file as well, throwing the warning because there were then two of them.